test/hotspot/jtreg/runtime/CommandLine/VMDeprecatedOptions.java

```diff
@@ -54,7 +54,6 @@
             {"InitialRAMFraction",        "64"},
             {"TLABStats",                 "false"},
             {"AllowRedefinitionToAddDeleteMethods", "true"},
-            {"ExtendedDTraceProbes", "true"},
```

Comment:
Do you want to add a comment that this flag cannot be tested at this time?

Reply:
Not really. It doesn't really make sense to me to comment something that isn't there - and then we'd have to remember to remove the comment again next release.